src/rngs/entropy.rs

```diff
@@ -48,6 +54,7 @@ use rngs;
 #[derive(Debug)]
 pub struct EntropyRng {
     source: Source,
+    _dummy: Rc<()>  // enforce !Send
```

Comment:
It's probably cleaner to use `core::marker::PhantomData` here.

Reply:
Yes, probably. I'm not sure that `Rc<()>` is a zero-sized type.

Reply:
Better to not rely on Rc just for !Send, raw pointers are not send: https://play.rust-lang.org/?gist=e350c6d98704bae32e0feb593687bf62&version=stable&mode=debug&edition=2015